TRANSFORMERS/VECTOR_MANIPULATION/VECTOR_INDEXING/VECTOR_INDEXING.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+from flojoy import flojoy, Vector, Scalar
+
+
+@flojoy
+def VECTOR_INDEXING(
+    default: Vector,
+    index: int = 0,
+) -> Scalar:
+    """The VECTOR_LENGTH node returns the length of the input
+
+    Inputs
+    ------
+    v : Vector
+        The input vector to find the length of.
+
+    Parameters
+    ----------
+    index : int
+        The index of the vector to return.
+
+    Returns
+    -------
+    Scalar
+        The scalar index of the input vector.
+    """
+
```

Comment:
Same suggestion as above, we should check for index out of bound, negative index, etc, and throw our own custom error message for a better user experience. User may be confused seeing the default whatever error msg from python/numpy